### PR TITLE
Repo Gardening: release version 1.1.0

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

﻿This should allow us to start using a stable version of the action in other repos. See the conversation in https://github.com/Automattic/vaultpress/pull/113 to find out more.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* The action should keep working on this PR.

### CODE REVIEW

File: projects/github-actions/repo-gardening/CHANGELOG.md
Comment (anomiex) [inline projects/github-actions/repo-gardening/CHANGELOG.md:24]: How'd this happen? Is it a bug in changelogger, or did you accidentally release 1.1.0-beta then manually edit?
```suggestion
[1.1.0]: https://github.com/Automattic/action-repo-gardening/compare/v1.0.0...v1.1.0
```

Comment (jeherve) [reply projects/github-actions/repo-gardening/CHANGELOG.md:24]: Yes, that's exactly that, my bad. I used `-b` without meaning to.

**Comment by jeherve** [comment]
Putting this on hold pending #19363.

**Comment by anomiex** [comment]
Note when you merge master into this after merging #19363, you'll want to do `changelogger write --amend` to include the changelog file from that PR in the release.

**Review by anomiex — APPROVED** [review]

**Comment by jeherve** [comment]
And with the autotagger job completed, we now have a new release 🎉 
https://github.com/Automattic/action-repo-gardening/releases/tag/v1.1.0

**Pull request merged**
